modules/op-configuring-pull-request-capabilities-in-GitHub-interceptor.adoc

```diff
@@ -0,0 +1,13 @@
+// Ths module is included in the following assembly:
+//
+// *cicd/pipelines/creating-applications-with-cicd-pipelines.adoc
+
+:_content-type: CONCEPT
+[id="op-configuring-pull-request-capabilities-in-GitHub-interceptor_{context}"]
+= Configuring pull request capabilities in GitHub Interceptor
+
+With a GitHub Interceptor, you can create logic that validates and filters GitHub webhooks. For example, you can validate the webhook’s origin and filter incoming events based on specified criteria. When you use GitHub Interceptor to filter event data, specify the event types that Interceptor has to accept in a field. 
+In {pipelines-title}, you can use the following capabilities of GitHub Interceptor:
```

Comment:
The first instance uses "a GitHub Interceptor" but the others leave out the "a" and just say "GitHub Interceptor", maybe make this consistent?